docs/modules/ROOT/pages/admin-api-reference.adoc

```diff
@@ -63,7 +64,7 @@ DATA='{
   "title": "Upgrade to v2",
   "description": "Upgrading contract to version 2.0",
   "type": "upgrade",
-  "metadata": { "newImplementation": "0x3E5e9111Ae8eB78Fe1CC3bb8915d5D461F3Ef9A9" }
+  "metadata": { "newImplementation": "0x3E5e9111Ae8eB78Fe1CC3bb8915d5D461F3Ef9A9", "newImplementationAbi": "..." }
```

Comment:
Now that you mention it, it would definitely be nice to add a realistic example (at least a string that qualifies as a valid ABI). Definitely not blocking this PR, but would be nice

Reply:
The reasoning was that the property name was obvious and I didnt want to take up unnecessary space with ABI. I'll create a minimalist example

Reply:
Thanks for adding it @dylankilkenny
Now that I look at it again, I think it makes a lot of sense since it's string encoded, and with cURL examples it's always a mess.